src/main/kotlin/com/team4099/robot2023/subsystems/drivetrain/swervemodule/SwerveModuleIO.kt

```diff
@@ -63,102 +64,107 @@ interface SwerveModuleIO {
 
     override fun toLog(table: LogTable?) {
       table?.put("driveAppliedVoltage", driveAppliedVoltage.inVolts)
-      table?.put("steeringAppliedVoltage", steeringAppliedVoltage.inVolts)
-      table?.put("driveStatorCurrentAmps", driveStatorCurrent.inAmperes)
-      table?.put("driveSupplyCurrentAmps", driveSupplyCurrent.inAmperes)
-      table?.put("steeringStatorCurrentAmps", steeringStatorCurrent.inAmperes)
-      table?.put("steeringSupplyCurrentAmps", steeringSupplyCurrent.inAmperes)
-      table?.put("drivePositionMeters", drivePosition.inMeters)
-      table?.put("steeringPositionRadians", steeringPosition.inRadians)
-      table?.put("driveVelocityMetersPerSecond", driveVelocity.inMetersPerSecond)
-      table?.put("steeringVelocityRadiansPerSecond", steeringVelocity.inRadiansPerSecond)
-      table?.put("driveTempCelcius", driveTemp.inCelsius)
-      table?.put("steeringTempCelcius", steeringTemp.inCelsius)
+      table?.put("swerveAppliedVoltage", steerAppliedVoltage.inVolts)
+      table?.put("statorCurrentDrive", statorCurrentDrive.inAmperes)
+      table?.put("supplyCurrentDrive", supplyCurrentDrive.inAmperes)
+      table?.put("statorCurrentSteer", statorCurrentSteer.inAmperes)
+      table?.put("supplyCurrentSteer", supplyCurrentSteer.inAmperes)
+      table?.put("drivePosition", drivePosition.inMeters)
+      table?.put("steerPosition", steerPosition.inRadians)
+      table?.put("steerTemp", steerTemp.inCelsius)
+      table?.put("driveTemp", driveTemp.inCelsius)
+      table?.put("driveVelocity", driveVelocity.inMetersPerSecond)
+      table?.put("steerVelocity", steerVelocity.inRadiansPerSecond)
```

Comment:
add units for everything here. `steerVelocityInDegeresPerSecond` or something

Reply:
done :)